Firestore/Source/Local/FSTLocalDocumentsView.mm

```diff
@@ -110,37 +114,37 @@ - (FSTDocumentDictionary *)documentsMatchingDocumentQuery:(const ResourcePath &)
 }
 
 - (FSTDocumentDictionary *)documentsMatchingCollectionQuery:(FSTQuery *)query {
-  // Query the remote documents and overlay mutations.
-  // TODO(mikelehen): There may be significant overlap between the mutations affecting these
-  // remote documents and the allMutationBatchesAffectingQuery mutations. Consider optimizing.
+  // Get the remote documents in the state in which the backend last acknowledged them.
```

Comment:
This comment and the one below are very close to merely restating what the code is doing. You might consider just dropping them.

Reply:
Done.